config/templates/cloud-config-controller

```diff
@@ -344,7 +344,7 @@ write_files:
           - --tls-private-key-file=/etc/kubernetes/ssl/apiserver-key.pem
           - --client-ca-file=/etc/kubernetes/ssl/ca.pem
           - --service-account-key-file=/etc/kubernetes/ssl/apiserver-key.pem
-          - --runtime-config=extensions/v1beta1/networkpolicies=true
+          - --runtime-config=extensions/v1beta1/networkpolicies=true, batch/v2alpha1
```

Comment:
@pieterlange I'll remove the space, don't remember where I saw an example using the space. Do you think it's also better to use `batch/v2alpha1=true` for consistency?